Anlab.Mvc/Pages/using-the-lab.md

```diff
@@ -43,7 +43,7 @@ title: "Using the Lab"
   <p>The samples are tested using standard methods as described in the section titled Methods of Analyses. At least ten percent of the samples are tested in duplicate at no cost as part of our quality assurance program. We also run at least one standard reference material sample with each set of analyses. These QC results are included in the final report. A description of the Lab's quality assurance and quality control measures are described in the section titled <a href="/pages/qa-qc">QA/QC</a>.</p>
 
   <h3><a name="five"></a>Reporting the Results:</h3>
-  <p>A link to the final report is e-mailed to the client when testing is complete. Reports are in a zip file containing the official pdf and an editable Excel file for your convenience. A copy will also be sent to the individual specified in the Work Order. Please review the results upon receipt of the report. For questions regarding the report, please contact the Lab (530-752-0147).</p>
+  <p>A link to the final report is e-mailed to the client when testing is complete. After April 11, 2023, reports are in a zip file containing the official pdf and an Excel file for your convenience. A copy will also be sent to the individual specified in the Work Order. Please review the results upon receipt of the report. For questions regarding the report, please contact the Lab (530-752-0147).</p>
```

Comment:
@benningd54 Perhaps "Reports for orders completed after April...."

Reply:
Yes, I see your suggestion.
I like it.
However, I don't know how to put it into effect at this stage of the process.